geopandas/base.py

```diff
@@ -283,7 +631,25 @@ def cascaded_union(self):
     @property
     def unary_union(self):
         """Returns a geometry containing the union of all geometries in the
-        ``GeoSeries``."""
+        ``GeoSeries``.
+
+        Examples
+        --------
+
+        >>> from shapely.geometry import Point
+        >>> s = geopandas.GeoSeries([Point(i, 0).buffer(0.7) for i in range(5)])
+        >>> s
+        0    POLYGON ((0.70000 0.00000, 0.69663 -0.06861, 0...
+        1    POLYGON ((1.70000 0.00000, 1.69663 -0.06861, 1...
+        2    POLYGON ((2.70000 0.00000, 2.69663 -0.06861, 2...
+        3    POLYGON ((3.70000 0.00000, 3.69663 -0.06861, 3...
+        4    POLYGON ((4.70000 0.00000, 4.69663 -0.06861, 4...
+        dtype: geometry
+
+        >>> union = s.unary_union
+        >>> union
```

Comment:
This is a potentially less future-friendly example, since it reflects underlying geometry implementation.  In this case, it also obscures the output.

How about showing truncated output?
```
>>> print(union)
POLYGON ((0.5000000000000001 -0.4894302343731161, 0.4949747468305837 -0.4949747468305829...
```

This example may also benefit from simpler geometries to make it more clear and obviate need to truncate WKT:
```
from shapely.geometry import box
s = geopandas.GeoSeries([box(0,0,1,1), box(0,0,2,2)])
union = s.unary_union
>>> print(union)
POLYGON ((0 0, 0 1, 0 2, 2 2, 2 0, 1 0, 0 0))
```